client/src/SignUpShopper.js

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import Api from './Api';
+
+//to add new component, make new file
+//to create (crud), use forms
+function SignUpShopper() {
+
+    //:id from Sections.js 
+    const {id} = useParams();
+    const history = useHistory();
+    
+    //initial value of section (in this case, obj with name, slug, and position attributes)
+    const [shopper, setShopper] = useState({
+        UserId: 0,
+        zipcode: 0
+    });
+
+    const [user, setUser] = useState({
+      photo: '',
+      birthday: 0,
+      gender: ''
+    });
+
+    //side effects, don't directly interact with output, don't refresh when it changes
+    useEffect(function(){
+        if(id){
+            Api.shoppers.get(id).then((response) => setShopper(response.data));
+            Api.users.get(id).then((response) => setUser(response.data));
+        }
+    }, []);
+
+    function onChange(event) {
```

Comment:
Mixing both user and shopper together means that you'll end up putting data for user in shopper and vice versa.

I'd recommend creating separate onChange functions- one for user, and one for shopper, and put the correct function with the appropriate input.